lib/class-factory.js

```diff
@@ -1950,6 +1951,57 @@ function ClassFactory (vm) {
         env.checkForExceptionAndThrowIt();
       }
 
+      if (spec.superClass) {
+        Object.defineProperty(Klass.$classWrapper.prototype, '$super', {
+          enumerable: true,
+          get: function () {
+            const superInstance = factory.cast(this, superClass);
+            return new Proxy(superInstance, {
+              get: function (target, property, receiver) {
+                const prop = superInstance[property];
+                if (!prop || !prop.overloads) {
+                  return prop;
+                }
+                function proxy (method) {
+                  return new Proxy(method, {
+                    apply: function (target, thisArg, args) {
+                      const tid = Process.getCurrentThreadId();
+                      try {
+                        method[PENDING_CALLS].add(tid);
+                        return method.apply(superInstance, args);
+                      } finally {
+                        method[PENDING_CALLS].delete(tid);
+                      }
+                    }
+                  });
+                }
+                return new Proxy(prop, {
+                  apply: function (target, thisArg, args) {
+                    const method = prop.overloads.find(m => m.canInvokeWith(args));
```

Comment:
I believe `Array.prototype.find` is missing in the Duktape runtime, so this code presumably crashes there?

Reply:
Yes, you are right. But i wonder why test code does not crash, maybe it does not use Duktape runtime. I will change this code.

Reply:
Fixed in lastest commit.